include/sender_cadmium.hpp

```diff
@@ -40,14 +60,21 @@ struct sender_defs {
     };
 };
 
+/** 
+* The Sender class sends out messages and receives acknowledges.
+*/
 template<typename TIME>
 class Sender {
-    using defs = sender_defs; // putting definitions in context
+    // putting definitions in context
```

Comment:
I understand that it is a single line comment and hence // is used. However, to maintain uniformity, I would suggest to use the same pattern of Doxigen notation throughout the file.

Reply:
Done